client/src/utils.ts

```diff
@@ -13,11 +13,11 @@ export function getUnixTime(date: Date, time: Date) {
     ).getTime() / 1000
   );
 }
-const UNIX_HOUR = 3600 * 1000;
+const UNIX_HOUR = 3600;
```

Comment:
Is UNIX_HOUR still used elsewhere? Otherwise delete. And why should it not be * 1000?